compiler/semobjconstr.nim

```diff
@@ -11,6 +11,8 @@
 
 # included from sem.nim
 
+from std/sugar import dup
```

Comment:
Now breaks the bsd cis... We need a better solution.

Reply:
where can i see the logs?

Reply:
see https://github.com/nim-lang/Nim/pull/17801